src/GeniusYield/Types/TxMetadata.hs

```diff
@@ -0,0 +1,111 @@
+module GeniusYield.Types.TxMetadata (
+  GYTxMetadata (..),
```

Comment:
Don't expose constructor.

Reply:
But the `GYTxMetadata` type constructor is used in line 330 of `src/GeniusYield/TxBuilder/Class.hs`.

Reply:
> But the GYTxMetadata type constructor is used in line 330 of src/GeniusYield/TxBuilder/Class.hs

I am talking about data constructor.

Reply:
And regarding, 

> Moreover, if I try to just expose GYTxMetadata (without "(..)") I get the error "The data constructor ... of newtype ‘GYTxMetadata’ is not in scope".

Prefer fromApi/toApi functions.

Reply:
Previous `metadataFromApi`, `metadataToApi` are now `fromApi`, `toApi` .

Reply:
Hey Antonio, I didn't meant to rename `metadataToApi` to `toApi` but I simply meant that if one wants to convert `GYTxMetadata` to `Api.TxMetadata` then instead of exposing the data constructor, one should make use of toApi like function such as your's `metadataToApi`.